docs/src/components/main.tsx

```diff
@@ -42,26 +271,32 @@ const styles = stylex.create({
   root: {
     display: 'flex',
     flexDirection: 'column',
-    gap: 120,
-    paddingTop: 98,
   },
-  mainRoot: {
+  title1: {
```

Comment:
Hello! Thank you so much for contributing to the document.

I have one suggestion, so I’m leaving a comment. The text “Declarative Overlay Pattern” overlaps with the background, which makes it difficult to read.

There might be several ways to fix this, but I think the easiest approach is to add an outline to the text using CSS like the example below. What do you think about adding an outline in this way?

If you have any better ideas, please feel free to share!

Reply:
```js
textShadow:
  '1px 1px 0 rgba(0, 0, 0, 0.1), -1px 1px 0 rgba(0, 0, 0, 0.1), 1px -1px 0 rgba(0, 0, 0, 0.1), -1px -1px 0 rgba(0, 0, 0, 0.1)',
```

Reply:
It's small difference! I want to suggest below shadow. a little bit color with black

### TO-BE textShadow: '0 0 2px rgba(0, 0, 120, 0.1), 0 0 5px rgba(120, 0, 120, 0.1), 0 0 12px rgba(0, 0, 120, 0.1)',
![image](https://github.com/user-attachments/assets/cfc3a09f-60cf-4280-80ab-1832459c1fb5)

### AS-IS textShadow: '0 0 2px rgba(0, 0, 120, 0.1), 0 0 5px rgba(120, 0, 120, 0.1), 0 0 12px rgba(0, 0, 120, 0.1)',
![image](https://github.com/user-attachments/assets/0d5aaca0-fca8-4351-ae7a-3024073d2086)

Reply:
Is this textShadow okay? [4d46700](https://github.com/toss/overlay-kit/pull/101/commits/4d4670074e4832b0d12e9c57b87a40530bcd80b8)

Reply:
I reflect your review! Thanks!